view/concept-shared.twig

```diff
@@ -96,7 +97,8 @@
           {% for language,labels in concept.allLabels(property.type) %}
             {% for value in labels %}
             <div class="row other-languages{% if loop.first %} first-of-language{% endif %}">
-            <div class="versal col-xs-6{% if value.type == "skos:altLabel" %} replaced{% endif %}">{% if value.containsHtml %}{{ value.label|raw }}{% else %}{{ value.label }}{% endif %}</div><div class="versal col-xs-6">{% if loop.first %} {{ language }}{% endif %}</div></div>
+                <div class="versal col-xs-6{% if value.type == "skos:altLabel" %} replaced{% endif %}">{% if value.containsHtml %}{{ value.label|raw }}{% else %}{{ value.label }}{% endif %}</div><div class="versal col-xs-6">{% if loop.first %} {{ language }}{% endif %}</div>
```

Comment:
What's the reason for this change? Seems to be just useless adjusting of whitespace, but hard to tell for sure.

Reply:
I was looking for a fancier indentation. Something according this: https://www.granneman.com/webdev/coding/formatting-and-indenting-your-html

Should I revert it to its original?

Reply:
Please revert. It's okay to clean up indentation, but let's keep that separate from changes in functionality if possible. In this case there was (AFAICT) no functional change in the template so this is just adding noise to the PR.

Reply:
Okay, I will do..

Reply:
Is it okay if I skip this now and I will take your recommendation on account from now on

Reply:
I had an issue with my IDE but now this is corrected